### DISCUSSION TITLE
Can FontAwesome be used?

### DISCUSSION THREAD

**Discussion by gldtn** [FelixKratz/SketchyBar#366] [Q&A] [answered]
I tried using font awesome but don't know if I'm setting the name wrong, but it doesn't seem to work. Would it be feasible to integrate it?

Thanks!

**Comment by FelixKratz** [comment, marked as answer]
Any font installed on your system (you can check via `Font Book` app) should be available to use in sketchybar. Make sure you specify the name such that it has `"<family>:<style>:<size>"` or use the `font.family`, `font.style` and `font.size` subdomains (https://felixkratz.github.io/SketchyBar/config/items#text-properties)
